modules/custom/az_card/src/Plugin/Field/FieldFormatter/AZCardDefaultFormatter.php

```diff
@@ -140,7 +140,7 @@ public function viewElements(FieldItemListInterface $items, $langcode) {
           if (!empty($parent_config['az_cards_paragraph_behavior'])) {
             $card_defaults = $parent_config['az_cards_paragraph_behavior'];
             // Is the card clickable?
-            if ($card_defaults['card_clickable']) {
+            if (isset($card_defaults['card_clickable']) && $card_defaults['card_clickable'] != FALSE) {
```

Comment:
Changing this back to the way it was for truthy testing to remove the PHPCS warning since @akslay checked the PHP docs:
>When converting to bool, the following values are considered false:
>the empty string, and the string "0"
https://www.php.net/manual/en/types.comparisons.php

```suggestion
            if (isset($card_defaults['card_clickable']) && $card_defaults['card_clickable']) {
```